iceoryx_hoofs/test/moduletests/test_vocabulary_string.cpp

```diff
@@ -2782,29 +2575,11 @@ TEST(String100, FindNotIncludedStringLiteralFails)
     EXPECT_THAT(res.has_value(), Eq(false));
 }
 
-TEST(String100, FindSTDStringInNotEmptyStringWorks)
-{
-    ::testing::Test::RecordProperty("TEST_ID", "794f62c2-deca-4511-9529-3353ff9ee552");
-    string<100U> testString("R2-D2");
-    std::string testStdString = "2";
-    auto res = testString.find(testStdString);
-    ASSERT_THAT(res.has_value(), Eq(true));
-    EXPECT_THAT(res.value(), Eq(1U));
-
-    res = testString.find(testStdString, 1U);
-    ASSERT_THAT(res.has_value(), Eq(true));
-    EXPECT_THAT(res.value(), Eq(1U));
-
-    res = testString.find(testStdString, 2U);
-    ASSERT_THAT(res.has_value(), Eq(true));
-    EXPECT_THAT(res.value(), Eq(4U));
-}
-
 TEST(String100, FindNotIncludedSTDStringFails)
 {
     ::testing::Test::RecordProperty("TEST_ID", "8b2116c9-5f7d-48b4-8c26-cb3b71cf0ea2");
     string<100U> testString("Kernfusionsbaby");
-    std::string testStdString = "abc";
+    string<100U> testStdString = "abc";
```

Comment:
Please revert the change and move this test to test_cxx_string_conversion.cpp

Reply:
Good catch, done!